File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n = len(nums)
        
        for i in range(n):
            for j in range(i+1, n):
                cal = nums[i] + nums[j]
                if cal == target: return [i,j]

File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int>vis;
        for (int i = 0; i < nums.size(); i++) {
            if (vis[target - nums[i]] > 0)
                return {vis[target - nums[i]] - 1, i};
            vis[nums[i]] = i + 1;
        }
        return {-1, -1};
    }
};